test/unit/features/filter/filter.spec.js

```diff
@@ -78,6 +78,41 @@ describe('Filters', () => {
     expect(vm.$el.textContent).toBe(String(1 / 4))
   })
 
+  it('handle division with parenthesis', () => {
+    const vm = new Vue({
+      data: { a: 20 },
+      template: `<div>{{ (a*2) / 5 | double }}</div>`,
+      filters: { double: v => v * 2 }
+    }).$mount()
+    expect(vm.$el.textContent).toBe(String(16))
+  })
+
+  it('handle division with dot', () => {
+    const vm = new Vue({
+      template: `<div>{{ 20. / 5 | double }}</div>`,
```

Comment:
made a quick test checking every single exception with `/[\w).\]\+\-\_$]/` maybe there is a more generic way to do this